lib/Command/AddCommand.php

```diff
@@ -0,0 +1,165 @@
+<?php
+
+declare(strict_types=1);
+/**
+ * @copyright Copyright (c) 2020 Vincent Petry <vincent@nextcloud.com>
+ *
+ * @license GNU AGPL version 3 or any later version
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+namespace OCA\Guests\Command;
+
+use OCA\Guests\GuestManager;
+use OCP\IUser;
+use OCP\IUserManager;
+use OCP\Mail\IMailer;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Question\Question;
+
+class AddCommand extends Command {
+	/** @var IUserManager */
+	private $userManager;
+	/** @var GuestManager */
+	private $guestManager;
+	/** @var IMailer */
+	private $mailer;
+
+	public function __construct(IUserManager $userManager, IMailer $mailer, GuestManager $guestManager) {
+		parent::__construct();
+		$this->userManager = $userManager;
+		$this->guestManager = $guestManager;
+		$this->mailer = $mailer;
+	}
+
+	protected function configure() {
+		$this
+			->setName('guests:add')
+			->setDescription('Add a new guest account')
+			->addArgument(
+				'created-by',
+				InputArgument::REQUIRED,
+				'User ID who is set as creator'
+			)
+			->addArgument(
+				'uid',
+				InputArgument::REQUIRED,
+				'User ID used to login (must only contain a-z, A-Z, 0-9, -, _ and @)'
+			)
```

Comment:
```suggestion
```

Reuse the email like done when using the web

Reply:
indeed: https://github.com/nextcloud/guests/blob/master/lib/Controller/UsersController.php#L138